server/routes/api/browser.py

```diff
@@ -60,3 +61,53 @@ def get_property_labels(dcid):
     """Returns all property labels given a node dcid."""
     labels = dc.get_property_labels([dcid]).get(dcid, {})
     return Response(json.dumps(labels), 200, mimetype='application/json')
+
+
+@cache.cached(timeout=3600 * 24, query_string=True)  # Cache for one day.
+@bp.route('/observationId')
```

Comment:
nit, the REST path is dash separated: /observation-id

Reply:
done